ui2/src/websocket/websocket.ts

```diff
@@ -54,8 +56,23 @@ function isErrorMessage(event: object): event is StreamErrors {
   return 'errors' in event;
 }
 
+// Decoder for a CreateEvent, copied from @daml/types
+const decodeCreateEvent = <T extends object, K, I extends string>(
+  template: Template<T, K, I>
+): jtv.Decoder<CreateEvent<T, K, I>> =>
+  jtv.object({
+    templateId: jtv.constant(template.templateId),
+    contractId: ContractId(template).decoder,
+    signatories: List(Party).decoder,
+    observers: List(Party).decoder,
+    agreementText: Text.decoder,
+    key: template.keyDecoder,
+    payload: template.decoder,
+  });
+
 function useDamlStreamQuery<T extends object, K, I extends string>(
   templateIds: string[],
+  templateMap: Map<string, Template<any, any, any>>,
```

Comment:
yup that works...wasn't sure if `T, K, I` made sense since to me that would refer to a specific `T`/`K`/`I` when the map would technically have all of them, but it works the same for `CreateEvent`. fixed.